packages/chat-client/src/constants/engine.ts

```diff
@@ -2,7 +2,7 @@ import { ONE_DAY, THIRTY_SECONDS } from "@walletconnect/time";
 import { RelayerTypes } from "@walletconnect/types";
 import { JsonRpcTypes } from "../types";
 
-export const KEYSERVER_URL = "https://keys.walletconnect.com";
+export const KEYSERVER_URL = "https://staging.keys.walletconnect.com";
```

Comment:
Can we use an environment variable for this ?

Reply:
This is going to be configurable pending spec change unless I'm mistaken, i'll get back to you